index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="media.css">
+
+    <title>Mobile Portfolio</title>
+</head>
+<body>
+        <header>
+        <nav>
+        <nav class="Main-container"><p class="logo"><a="">My Logo</a></p>
+        <nav class="menu"><i class="fa fa-bars"></i></nav>
+        <nav class="menu2">
+            <ul>
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+        </nav>
+        </nav>
```

Comment:
```suggestion
        <nav>
        <div class="Main-container"><a="logo">My Logo</a><div>
        <div class="menu"><i class="fa fa-bars"></i></div>
        <div class="menu2">
            <ul>
                <li>Portfolio</li>
                <li>About</li>
                <li>Contact</li>
            </ul>
       
        </nav>
```

Reply:
Kindly use single nav element to wrap the entire nav section

Reply:
i had used but reviewer asked me to remove all the div

Reply:
and put every thing in nav tags